src/plugins/auto/tests/unit/get_device_list.cpp

```diff
@@ -134,25 +137,29 @@ const std::vector<Params> testConfigs = {Params{" ", " "},
                                          Params{"CPU,VPUX", "CPU,VPUX"},
                                          Params{"CPU,-VPUX", "CPU"},
                                          Params{"CPU,-INVALID_DEVICE", "CPU"},
+                                         Params{"CPU,INVALID_DEVICE", "CPU"},
+                                         Params{"INVALID_DEVICE", ""},
```

Comment:
add {"INVALID_DEVICE,GPU,CPU", ""},{"INVALID_DEVICE,CPU", ""},

Reply:
This test case is just used for check the logic of filtering the valid devices from the inputting property `ov::device::priorities`. 
From my understanding,  `"GPU,CPU" `will be the expected final valid device candidate list for the case `"INVALID_DEVICE,GPU,CPU"` as the specified devices candidate list.
Could you help explain why the empty device list is returned for the mentioned 2 test cases.

Reply:
not empty, just add "INVALID_DEVICE,GPU,CPU" this kind of test

Reply:
Got it.